cohorts/load.py

```diff
@@ -1282,7 +1286,7 @@ def plot_joint(self, on, on_two=None, x_col=None, plot_type="jointplot", stat_fu
         jointplot_kwargs : dict, optional
```

Comment:
Should this be specific to `jointplot` or just have `plot_kwargs` passed to all the plots? i.e. passing `ax` might be useful